lib/mato/html_filters/syntax_highlight.rb

```diff
@@ -5,6 +5,12 @@
 module Mato
   module HtmlFilters
     class SyntaxHighlight
+      class RougeError < StandardError
+      end
+
+      def initialize(on_rouge_error: ->(ex) { warn ex })
```

Comment:
Rouge's error will be displayed to stderr by default.
User can configure it, for example, use `Rails.logger` instead, or send the error to Sentry.